scenario_runner/launch/scenario_runner.launch.xml

```diff
@@ -1,16 +1,15 @@
 <?xml version="1.0" ?>
 <launch>
-
     <arg name="map_path" default="/media/tier4/ExtremeSSD2TB/t4b/kashiwanoha"/>
-    <arg name="t4b_launch_path" default="$(find scenario_api)/launch/scenario.launch"/>
+    <arg name="t4b_launch_path" default="$(find scenario_api)/launch/scenario.launch.xml"/>
```

Comment:
We can probably delete the launch file in this PR, and integrate it into `launch.py` in the other PR.

Reply:
@nnmm I would say this should stay in case someone wants to launch a standalone scenario running but then it gets confusing if someone tries to use this with the planning launcher. Since I am a bit in between, I will vote for code removal as well.

Reply:
The alternative would be to keep it here, but in Python and with the process exit handler added. That's probably even nicer.